.travis.yml

```diff
@@ -2,19 +2,26 @@ sudo: false
 language: node_js
 node_js:
   - '6.10.1'
-script: make test-ci
-cache: yarn
+script:
+  - export IP_ADDR=$(ip addr | grep eth -A 4 | grep 'inet ' | awk '{ print $2 }' | sed 's/\/..//')
+  - pnpm run build
+  - pnpm run test-ci
+before_install:
+  - curl -L https://unpkg.com/@pnpm/self-installer | node
+install:
+  - pnpm recursive install --filter '@cycle/*'
+cache:
+  directories:
+    - /home/travis/.pnpn-store
```

Comment:
Typo here? `pnpn`

Reply:
I was already wondering why the caching did not seem to work _facepalm_